### ISSUE TITLE
Company Logo Not Showing in Paypal

### ISSUE THREAD

**Issue by Rickerby** [wojodesign/simplecart-js#155]
Hi,

I have just updated my cart from version 2.0 to use the latest simpleCart.js version 2.2
everything is working fine except the companies logo is now not appearing above the totals 
in the Paypal cart as it did with version 2.0 and has been replaced with the default email address 
of the Paypal account. 

I have checked the Paypal settings and nothing seems to have changed there, and everything is as it was
when the cart was using 2.0

I noticed that simpleCart.js used to have the line of code this.companyLogo "https://mycompanylogo.jpg"
and version 2.2 does not. Is this line of code now located somewhere else?

Any help on getting the logo to show up again would be hugely appreciated.

**Comment by brettwejrowski** [comment]
Hey Rickerby,  I am not seeing the line of code from v2 or v2.2.  Could it have been something extra added to the code?  You should be able to add this to the paypal checkout yourself by referencing the paypal docs.

**Comment by Rickerby** [comment]
Hi Brett,

You are right! It was a piece of custom code. Your response reminded me of how I first stumbled across simpleCart. I found a website that was using the cart and traced their files to the simpleCart website. 

After a comparing the version 2.0 simpleCart.js file to the version 2.2 simpleCart.js file I discovered the following line was appeared in version 2.0 "this.companyLogo = "https://imagehost.com/sslpic/123456.jpg" but not in version 2.2. Not knowing much about Javascript I modified the line to "me.companyLogo = "https://imagehost.com/sslpic/123456.jpg" and added it to simpleCart.js version 2.2 and the companies logo now appears in Paypal. Problem solved.
